priv/repo/migrations/20200904184516_alter_sessions.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Hexpm.RepoBase.Migrations.AlterSessions do
+  use Ecto.Migration
+
+  def change do
+    alter table(:sessions) do
```

Comment:
There will need to be a second pass to make all of these not nullable, but first this must go out and the table must be pruned in prod must be pruned before doing so.

Reply:
Can't we prune the table in the migration since all sessions will be invalidated anyway?

Reply:
We can but I generally don't do data migrations this way, but can oblige if you feel ok with it.